airflow/utils/log/file_task_handler.py

```diff
@@ -574,9 +574,9 @@ def _read_from_logs_server(self, ti, worker_log_rel_path) -> tuple[list[str], li
                 messages.append(f"Found logs served from host {url}")
                 logs.append(response.text)
         except Exception as e:
-            from httpx import UnsupportedProtocol
+            from requests.exceptions import InvalidSchema
```

Comment:
should be 
```python
from requests.exceptions import InvalidURL
```
otherwise the fix on EmptyOperator's log msg doesn't work https://github.com/apache/airflow/pull/35536

i'm going to open a new PR later if no one beats me to it